src/Dynamic/Includes/cards/taskcard.pug

```diff
@@ -0,0 +1,10 @@
+.card.is-box-shadow
+    .card-content
+        .media
+            .media-content
+                p.title.is-3 #{task.name}
+                p.subtitle.is-6
+                    | Next Run: #{task.nextInvocation}
```

Comment:
Could we also show the interval for the automated runs of the task?

Reply:
Its stored as a cron object with the nextInvocation which is not easy to convert back to the original interval. I will add this with a job structure rework.